src/common/perf_counters.cc

```diff
@@ -465,7 +424,7 @@ PerfCounters::PerfCounters(CephContext *cct, const std::string &name,
     m_upper_bound(upper_bound),
     m_name(name.c_str()),
     m_lock_name(std::string("PerfCounters::") + name.c_str()),
-    m_lock(m_lock_name.c_str())
+    m_lock(ceph::make_mutex(m_lock_name.c_str()))
```

Comment:
no need to pass `const char*`, `m_lock_name` would suffice. please see the constructor of `mutex_debug_impl`.

Reply:
sure.